### PR TITLE
Likes and history

### PR DESCRIPTION
**Pull request by ravikumar0403**
### Change log:
--- 
- Single video page
- Like/Dislike a video
- watch history
- delete single video from watch history
- clear watch history

### CODE REVIEW

**Review by yashpatel1797 — APPROVED** [review]
- Code looks clean
- functionality works fine.
- when the user is not logged in then on click of the like button there is noting happening you can give a toast or redirect to the login page.

**Comment by ravikumar0403** [comment]
> * Code looks clean
> * functionality works fine.
> * when the user is not logged in then on click of the like button there is noting happening you can give a toast or redirect to the login page.

Thanks @yashpatel1797  for bringing this to my notice. I'll make the change

**Comment by parthpandyappp** [comment]
Hey @ravikumar0403 
Your code looks great. Just to bring to your attention that while using vitube, I was able to add videos to watch later even if I wasn't authenticated.

_Ignore in case watch later only for authentication isn't actually a part of this PR and go ahead with merge_

**Review by parthpandyappp — APPROVED** [review]

File: src/components/PlaylistVideoCard.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+import { Link } from "react-router-dom";
 
-export const PlaylistVideoCard = ({ video }) => {
+export const PlaylistVideoCard = ({ video, deleteBtnHandler }) => {
   const { _id, title, creator, views } = video;
 
   return (
     <div className="card card card-horizontal playlist w-auto">
-      <div className="card-media">
+      <Link to={`/video/${video._id}`} className="card-media">
```
Comment (akshay-rajput) [inline src/components/PlaylistVideoCard.js:11]: title="watch videonamehere"

**Pull request merged**